lib/react_client.dart

```diff
@@ -27,6 +29,8 @@ export 'package:react/react_client/react_interop.dart'
 export 'package:react/react.dart'
     show ReactComponentFactoryProxy, ComponentFactory;
 
+@Deprecated('5.0.0')
```

Comment:
```suggestion
@Deprecated('6.0.0')
```

The version within the annotation should be the version in which the deprecated thing will be removed.